grc/blocks/grc.tree.yml

```diff
@@ -4,6 +4,10 @@
   - pad_sink
   - virtual_source
   - virtual_sink
+  - bus_sink
```

Comment:
Not sure if these lines need to be put back here but in adding back busports functionality, copied a bunch of code from 3.7